src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
Omit the `static assert (0);` if you want.

(Although, it's not just fluff - it is documenting the fact that this code depends upon the precise bit-ness of the types involved, and will probably break if the assumption that `size_t` is either 32- or 64-bit is ever broken on some future architecture.)

But, trying to change the style of a module by updating some random small piece of it just increases the chaos. I was already [asked not to do that](https://github.com/D-Programming-Language/druntime/pull/1452#issuecomment-170805228) on the very PR you're fixing. (Admittedly, the context was somewhat different.)